bot/exts/evergreen/cheatsheet.py

```diff
@@ -67,25 +73,29 @@ def result_fmt(self, url: str, body_text: str) -> t.Tuple[bool, t.Union[str, Emb
         aliases=("cht.sh", "cheatsheet", "cheat-sheet", "cht"),
     )
     @commands.cooldown(1, 10, BucketType.user)
-    async def cheat_sheet(
-            self, ctx: Context, *search_terms: str
-    ) -> None:
+    @with_role(Roles.everyone)
+    async def cheat_sheet(self, ctx: Context, *search_terms: str) -> None:
         """
         Search cheat.sh.
 
         Gets a post from https://cheat.sh/python/ by default.
         Usage:
         --> .cht read json
         """
-        url = f'https://cheat.sh/python/{quote_plus(" ".join(search_terms))}'
-
-        escape_tt = str.maketrans({"`": "\\`"})
-        ansi_re = re.compile(r"\x1b\[.*?m")
-
-        async with self.bot.http_session.get(url) as response:
-            result = ansi_re.sub("", await response.text()).translate(escape_tt)
-
-        is_embed, description = self.result_fmt(url, result)
+        category = utils.get(ctx.message.guild.categories, id=Categories.help_in_use)
+        all_allowed_channels = WHITELISTED_CHANNELS + tuple(category.channels)
```

Comment:
For the category check, you don't really have to fetch all the channels. You can instead check if `ctx.channel.id == Categories.help_in_use`